src/peft/utils/constants.py

```diff
@@ -72,6 +72,7 @@ def starcoder_model_postprocess_past_key_value(past_key_values):
     "mistral": ["input_layernorm", "post_attention_layernorm", "norm"],
     "phi": ["input_layernorm", "final_layernorm"],
     "gemma": ["input_layernorm", "post_attention_layernorm", "norm"],
+    "gemma2": ["input_layernorm", "norm"],
```

Comment:
Is it normal that they are not the same since you said that it was the same layers as gemma1 in the description of the PR.

Reply:
Not sure what I did there, I checked again and gemma2 has in fact more layer norm layers than gemma1, not fewer. I updated the targets accordingly.